runs/2021/pyladies-olomouc-podzim/info.yml

```diff
@@ -235,3 +235,38 @@ plan:
       - title: Úkoly (🧮)
         url: https://projekty.pyladies.cz/session?course=pyladies-2021-olomouc-podzim&session=class
         type: homework
+
+  - title: Závěrečný projekt
+    slug: asteroids
+    date: 2022-01-20
+    materials:
+      - lesson: projects/asteroids
+        type: lesson
+      - lesson: projects/snake
+        type: lesson
+      - title: Množinový tahák
```

Comment:
Why is the set cheatsheet included here? :thinking:

Reply:
Asteroids project uses/references `set`

Reply:
Aha, okay :)